Installer-Linux/linux-hosts-installer.sh

```diff
@@ -6,19 +6,31 @@
 # Copyright (c) 2017, 2018, 2019, 2020 Mitchell Krog - @mitchellkrogza
 # Copyright (c) 2017, 2018, 2019, 2020 Nissar Chababy - @funilrys
 
-#
-# root has to run the script
-#
-if [[ $(id -u -n) != "root" ]]
-    then
-    printf "You need to be root to do this!\nIf you have SUDO installed, then run this script with `sudo ${0}`"
-    exit 1
-fi
-
-# First Backup Existing hosts file
-sudo mv /etc/hosts /etc/hosts.bak
 
-# Now download the new hosts file and put it into place
-sudo wget https://hosts.ubuntu101.co.za/hosts -O /etc/hosts
-
-exit 0
+if [ $(id -u) -eq 0 ] ; then
+    # The 'real' script:
+    # First Backup Existing hosts file
+    mv /etc/hosts /etc/hosts.bak
+    # Now download the new hosts file and put it into place
+    wget https://hosts.ubuntu101.co.za/hosts -O /etc/hosts
+    exit 0
+else
+    # Multiple tricks to run this script as root before running the real script
+    echo "This script needs to be ran as root, let's switch to root:"
+    sudo $0 && exit 0
+    if [ $? -eq 127 ] ; then
+        echo "I can't find 'sudo', I'll try to use 'su' to become root."
+	echo "(Remember that 'su' is considerd deprecated (it needs a root password), I strongly recommend installing 'sudo'.)"
```

Comment:
tab -> 8 spaces
```suggestion
        echo "(Remember that 'su' is considerd deprecated (it needs a root password), I strongly recommend installing 'sudo'.)"
```

Reply:
I also noticed that i wrote _considerd_ instead of _consider**e**d_ . I'll fix both things in the next commit (after taking a look at  `pkexec` and including it if we can use it)